core/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestClusterAllocationExplainAction.java

```diff
@@ -78,6 +78,7 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
                     client.admin().cluster().allocationExplain(req, new RestBuilderListener<ClusterAllocationExplainResponse>(channel) {
                 @Override
                 public RestResponse buildResponse(ClusterAllocationExplainResponse response, XContentBuilder builder) throws Exception {
+                    builder.humanReadable(true);
```

Comment:
@clintongormley what do you think about setting the `human` readable flag for the REST API here?  it only affects the time fields (e.g. `remaining_delay` vs `remaining_delay_in_millis` for the delayed allocation scenario) and byte fields for shard stores.  Should the user just be required to put `?human` to get the more user friendly outputs similar to the other APIs?